src/components/Questions/QuestionModal.vue

```diff
@@ -232,10 +266,8 @@ export default defineComponent({
         props.hasQuizEnded ||
         !isQuizAssessment.value
       ) {
-        // emit an event if the requested question needs to be fetched
-        if (!isQuestionFetched(state.localCurrentQuestionIndex + 1)) {
-          context.emit("fetch-question-bucket", state.localCurrentQuestionIndex + 1)
-        }
+        // if already fetched, emit will have no effect
+        context.emit("fetch-question-bucket", state.localCurrentQuestionIndex + 1)
```

Comment:
didn't understand

Reply:
emit will have no effect because if the current question index + 1 is fetched, then isQuestionFetched (in Player.vue) will return True.